providers/implementations/signature/dsa.c

```diff
@@ -50,11 +53,50 @@ typedef struct {
     OPENSSL_CTX *libctx;
     DSA *dsa;
     size_t mdsize;
+    unsigned int flag_allow_md : 1;
```

Comment:
Maybe put the reason for this flag as a comment (like you do when you actually use it)

Reply:
Will do

Reply:
Done